Meetups.elm

```diff
@@ -4,7 +4,7 @@ import Speakers exposing (..)
 
```

Comment:
in Speakers.elm, you could add
```elm
type alias Speakers = (Speaker, List Speaker)
```
or even something like
```elm
type alias Speakers =
  { main : Speaker
  , featuring : List Speaker
  }
```
which is a tad more verbose (matter of taste I guess)